src/Microsoft.AspNetCore.Http.Features/IResponseCookiesFeature.cs

```diff
@@ -3,8 +3,33 @@
 
 namespace Microsoft.AspNetCore.Http.Features
 {
+    /// <summary>
+    /// Feature containing cookies which will be returned with the response.
+    /// </summary>
     public interface IResponseCookiesFeature
     {
+        /// <summary>
+        /// Initial capacity of instances used in the implementation of this feature, in <see cref="char"/>s.
+        /// </summary>
+        /// <remarks>
+        /// For example, the initial capacity of <see cref="System.Text.StringBuilder"/> instances obtained
+        /// from an object pool.
+        /// </remarks>
+        int InitialPooledInstanceCapacity { get; set; }
+
+        /// <summary>
+        /// Maximum retained capacity of instances used in the implementation of this feature, in <see cref="char"/>s.
+        /// Instances larger than this will be deleted rather than preserved in the pool.
+        /// </summary>
+        /// <remarks>
+        /// For example, the maximum retained capacity of <see cref="System.Text.StringBuilder"/> instances obtained
+        /// from an object pool.
+        /// </remarks>
+        int MaximumRetainedPooledInstanceCapacity { get; set; }
```

Comment:
Copying @Tratcher's comment from previous PR: :-1: This is an implementation detail, not a generic feature property.